src/View.php

```diff
@@ -121,7 +121,7 @@ class View extends AbstractView implements JsExpressionable
      */
     public $element;
 
-    /** @var ExecutorFactory */
+    /** @var string Seed class name */
```

Comment:
No! We do not want static factory, we want factory instance.

Reply:
I will try to get there. hold on ;-)
This is the plan,  Let me know if this is ok:
- View::getExecutorFactory() will return a Singleton instance;
   - Either from View::$excutorProperty class name or using app default

I still need to add a few things in the factory the I will try to get this working.

Reply:
I think this way:

- app instance executor factory
- View `excutorProperty` property is either null - factory from app is used, or an instance, which is used directly